charts/datadog/values.yaml

```
@@ -100,6 +100,11 @@ datadog:
   #   - "<KEY_1>:<VALUE_1>"
   #   - "<KEY_2>:<VALUE_2>"
 
+  # kubelet configuration
+  kubelet:
+    # datadog.kubelet.kubelet_host -- Override kubelet IP
+    kubelet_host:  # spec.nodeName
```

maybe `host` is enough

```suggestion
    host:  ""
```